src/main/msp/msp.c

```diff
@@ -3337,6 +3338,7 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
         governorConfigMutable()->gov_rpm_filter = sbufReadU8(src);
         governorConfigMutable()->gov_tta_filter = sbufReadU8(src);
         governorConfigMutable()->gov_ff_filter = sbufReadU8(src);
+        governorConfigMutable()->gov_spoolup_min_throttle = sbufReadU8(src);
```

Comment:
For keeping compatibility with 2.0, this must be in if (sbufBytesRemaining(src) >= 1)

Reply:
That makes sense, updated the PR.